libqtile/widget/wlan.py

```diff
@@ -42,13 +44,24 @@ def __init__(self, **config):
         self.add_defaults(Wlan.defaults)
 
     def poll(self):
-        interface = Wireless(self.interface)
         try:
-            stats = Iwstats(self.interface)
-            quality = stats.qual.quality
-            essid = interface.getEssid()
-            return "{} {}/70".format(essid, quality)
-        except IOError:
-            logging.getLogger('qtile').error('%s: Probably your wlan device '
-                    'is switched off or otherwise not present in your system.',
+            interface = Wireless(self.interface)
+            try:
+                stats = Iwstats(self.interface)
+                quality = stats.qual.quality
+                essid = interface.getEssid()
+                return "{} {}/70".format(essid,quality)
```

Comment:
Watch the pep8 check here, the Travis tests are failing because you need a space after the comma